localization/english/wc_traits_magic_lifestyles_l_english.yml

```diff
@@ -0,0 +1,204 @@
+
+ trait_wc_lifestyle_elemental_fire_magic:0 "Fierce Pyromancers"
+ trait_user_of_elemental_fire_magic_1:0 "Novice Pyromancer"
+ trait_user_of_elemental_fire_magic_1_desc:0 "If a threat stands before Pyromancers, the answer is always to neutralize it in a glorious blaze. A lifetime dedicated to the study of fiery forces only fuels their pyromania, and they take pride—even pleasure—in igniting their enemies in wild bursts of flame. Mistake their affinity for watching things burn as a lack of self-control at your own peril."
+ trait_user_of_elemental_fire_magic_1_character_desc:0 "[ROOT.GetCharacter.GetFirstNameNoTooltip] is a $trait_user_of_elemental_fire_magic_1$. $trait_user_of_elemental_fire_magic_1_desc$"
+ trait_user_of_elemental_fire_magic_2:0 "Journeyman Pyromancer"
+ trait_user_of_elemental_fire_magic_2_desc:0 "$trait_user_of_elemental_fire_magic_1_desc$"
+ trait_user_of_elemental_fire_magic_2_character_desc:0 "[ROOT.GetCharacter.GetFirstNameNoTooltip] is a $trait_user_of_elemental_fire_magic_2$. $trait_user_of_elemental_fire_magic_1_desc$"
+ trait_user_of_elemental_fire_magic_3:0 "Expert Pyromancer"
+ trait_user_of_elemental_fire_magic_3_desc:0 "$trait_user_of_elemental_fire_magic_1_desc$"
+ trait_user_of_elemental_fire_magic_3_character_desc:0 "[ROOT.GetCharacter.GetFirstNameNoTooltip] is an $trait_user_of_elemental_fire_magic_3$. $trait_user_of_elemental_fire_magic_1_desc$"
+ trait_user_of_elemental_fire_magic_4:0 "Master Pyromancer"
+ trait_user_of_elemental_fire_magic_4_desc:0 "$trait_user_of_elemental_fire_magic_1_desc$"
+ trait_user_of_elemental_fire_magic_4_character_desc:0 "[ROOT.GetCharacter.GetFirstNameNoTooltip] is a $trait_user_of_elemental_fire_magic_4$. $trait_user_of_elemental_fire_magic_1_desc$"
+ trait_user_of_elemental_fire_magic_5:0 "Legendary Pyromancer"
+ trait_user_of_elemental_fire_magic_5_desc:0 "$trait_user_of_elemental_fire_magic_1_desc$"
+ trait_user_of_elemental_fire_magic_5_character_desc:0 "[ROOT.GetCharacter.GetFirstNameNoTooltip] is a $trait_user_of_elemental_fire_magic_5$. $trait_user_of_elemental_fire_magic_1_desc$"
+
+ trait_wc_lifestyle_elemental_water_magic:0 "Adaptable Hydromancers"
+ trait_user_of_elemental_water_magic_1:0 "Novice Hydromancer"
+ trait_user_of_elemental_water_magic_1_desc:0 "Hydromancers have a serene affinity for water—they do not necessarily seek or turn to the divine, yet they feel a profound spiritual connection with the source from which all mortal life took root."
+ trait_user_of_elemental_water_magic_1_character_desc:0 "[ROOT.GetCharacter.GetFirstNameNoTooltip] is a $trait_user_of_elemental_water_magic_1$. $trait_user_of_elemental_water_magic_1_desc$"
+ trait_user_of_elemental_water_magic_2:0 "Journeyman Hydromancer"
+ trait_user_of_elemental_water_magic_2_desc:0 "$trait_user_of_elemental_water_magic_1$"
```

Comment:
Wrong loc.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/124265575-ea4c6180-db46-11eb-89b5-d92a16249265.png)

</details>

Reply:
Should be fixed, because I reverted these changes

Reply:
> Should be fixed, because I reverted these changes

The issue is still there.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/127191528-62300b39-5461-4562-a592-ea365f8799a1.png)

</details>

Reply:
Ah, got it
Should be fixed now